### DISCUSSION TITLE
error with ory using simple react.js

### DISCUSSION THREAD

**Discussion by anovazzi1** [ory/kratos#2953] [Q&A]
``Hi, I'm working on a project that already have ory/kratos, but now we are meaking a new front-end with react instead of angular and I wanted to keep the ory/kratos auth. But i follow the quickstart for reactJs and changed my config file, but now i can only redirect to the login page and get back to my react app, but i don't get the user data from the API.
I'm getting this errors:
`Access to XMLHttpRequest at 'http://localhost:4455/sessions/whoami' from origin 'http://localhost:3000' has been blocked by CORS policy: The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'. The credentials mode of requests initiated by the XMLHttpRequest is controlled by the withCredentials attribute.
`
`xhr.js:169          GET http://localhost:4455/sessions/whoami net::ERR_FAILED 404`
![image](https://user-images.githubusercontent.com/70824914/207744143-0c50b95b-5f76-4b2f-89d4-a8f592e6b58d.png)

As you can see I'm sending the cookies but I don't know why doesn't work
this is my config file:
![image](https://user-images.githubusercontent.com/70824914/207744799-8e43cb3c-d630-4772-9539-d0fca9e7228e.png)

**Comment by MJENDIDS** [comment]
@anovazzi1 I am having the same issue, did you end up fixing it?

**Reply by anovazzi1** [reply]
almost, i found out that because of docker mt url for whoami is not in the same port, so i'm making the authentication with axios end the rest api.
But now i'm facing some problems with cors

**Reply by MJENDIDS** [reply]
I am having the exact same issue right now, https://github.com/ory/kratos/discussions/2970, do you think it could be a bug? Their sdk does not even work for me.

**Reply by anovazzi1** [reply]
I really don't know i bypassed that by changing some proxy configs on my node.package file and found some conflicts on the Oathkeeper

**Comment by MJEND7** [comment]
The issue I had was that version 0.11.0 did not have the correct cors setup in the source code(https://github.com/ory/kratos/pull/2934), I would recommend that you build you own custom version 0.11.0 after the PR(https://github.com/ory/kratos/pull/2934) was pushed, or roll back to 0.10.1. Hope this helped, sorry for the late response, got busy :)
